src/data_acquisition.py

```diff
@@ -0,0 +1,81 @@
+import argparse
```

Comment:
Considered adding docstrings to this file to make the functions clearer

Reply:
Thanks! Docstrings have been added. I will commit the changes later today.